hydrolib/core/io/dflowfm/structure/models.py

```diff
@@ -108,27 +108,62 @@ def check_location(cls, values: dict) -> dict:
         filtered_values = {k: v for k, v in values.items() if v is not None}
         structype = filtered_values.get("type", "").lower()
 
-        if not (structype == "compound" or issubclass(cls, (Compound, Dambreak))):
+        if not (structype == "compound" or issubclass(cls, (Compound))):
```

Comment:
Please suggest the following for readibility:
```suggestion
        if structype == "compound" or issubclass(cls, (Compound)):
            return values
```

And re-indent the code below.

Reply:
Good idea, I committed this in 2f7b97d.